README.md

```diff
@@ -140,6 +140,39 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Chef Habitat supervisor during the install.
 
+### Airgapped Installation (Recommended prep work, if airgapped)
```

Comment:
I'm not sure we are ready to put in Airgap instructions yet until the installation actually supports it (ie, the install script runs and completes without an Internet connection?).

Reply:
Our file structure is pretty complicated. I think it is time for me to add a local documentation server & structure. Let's keep the airgap here for _today_. Once merged, I can put it onto a draft page.

Reply:
@chefsalim I'm very pleased to assure you that the changes in this PR accomplish the following:
* Preparation work enumerated in detail for an Airgap Installation.
* Once prep-work is completed, the Airgap install with no Internet connection, has been validated multiple times to be working successfully.
* Install _with_ Internet connection has been validated to work just as before.

Reply:
Excellent, that is great to know @jeremymv2!